### DISCUSSION TITLE
【奇忽局club#13】驰骋在比特荒野上的岛屿与河流——漫谈去中心化城市X车联网【🎤录音房】

### DISCUSSION THREAD

**Discussion by Lupingyue** [qihuju/qihuju.github.io#29] [Show and tell]
【時間】2021年4月9日，週五
北京時間：8pm-10pm
倫敦時間：1pm-3pm
美東時間：8am-10am
美西時間：5am-7am

【話題】
本期话题源于 @TS 【不确定性：河流】和【确定性：岛屿】的二元化比喻 #23 
“这个岛屿河流的二元化比喻最早来自于我对未来城市的想象。英国建筑师在60年代时候提出过一个Walking City的构想，有点像是哈尔的移动城堡的放大版或者是流浪地球的缩小版……更有可能的未来也许会出现在自动驾驶技术达到L5之后。这意味着物联网，车联网的底层通讯架构都已经成熟。那么未来人类的栖息地就会由岛屿+河流交织起来……这里的河流包含了人流，物流，体验流和信用流……那么已经从【拥有】过渡到【调用】逻辑的车联网就编织了上述flow之间与虚拟世界和物理世界肌理……一个去中心化的walking city = walking room……这实际上跟人类游牧状态下的生活方式很类似”——@TS

【流程】
1，@TS 进行主题分享「30分钟」
图文资料分享在奇忽局微信群
2，@Mingxuan ✖️ @TS “从建筑电讯派、比特之城，到比特荒野的岛屿与河流”「20分钟」
3，提问环节 
4，自由讨论

以下為建議名單，有空就來，沒空歡迎關注腦圖及群里的討論；）
【moderators】
@TS
@李大維 
@Mingxuan 
@Tz  
@Miki Dai 
@Andy Liu 
@HUI
@In_K
@阿呆 🦁 
@楊梓萌 Brendan(醒著造夢） 
@岳路平

协同编辑：https://mm.tt/1847893198?t=PvqSFYJAeR

我们是比特荒野上的殖民者，也是指路人；我们是赛博朋克原住民的血族，也是他们的导师；我们回溯往矣，我们拥抱当下，我们探索未来；我们是奇忽局。
We are colonists in the Byte Wilderness, and guides; we are the blood of the Cyberpunk Natives, and mentors; we are looking after the past, we are embracing the present, we are exploring the future; we are the Singularity Fantasy Bureau.
                                                     ——Tz

**Comment by Lupingyue** [comment]
> 
> 【流程】
> 1，@ts 进行主题分享「30分钟」
> 图文资料分享在奇忽局微信群
> 2，@mingxuan ✖️ @ts “从建筑电讯派、比特之城，到比特荒野的岛屿与河流”「20分钟」

> 
> 协同编辑：https://mm.tt/1847893198?t=PvqSFYJAeR
> 

@cyberlinden @MingxuanXie ,有什么相关的链接、书目、影片……，有空可以进入脑图编辑，如果嫌麻烦也可以在这里贴出来，我来转到脑图里。

**Reply by midai17** [reply]
是否适合在周五活动结束之后趁热打铁，正式在微信群和俱乐部等处宣发下“奇忽局宣言1.0”?

**Reply by Lupingyue** [reply]
> 是否适合在周五活动结束之后趁热打铁，正式在微信群和俱乐部等处宣发下“奇忽局宣言1.0”?

可以啊。你可以把“奇忽局宣言1.0”編輯一下嗎？我覺得不求成熟，而是不斷尋求大家的參與和迭代。 #12

**Reply by dadabit** [reply]
OK

**Reply by Lupingyue** [reply]
> OK

欢迎👏👏，是Dada，还是Dada daddy😄

**Reply by dadabit** [reply]
It’s dada

**Reply by dadabit** [reply]
这里好壮观啊！

**Reply by Lupingyue** [reply]
> It’s dada

欢迎dada，早上好
欢迎开启你的project，或者参与其他人发起的project

**Reply by dadabit** [reply]
👌👌

**Reply by Lupingyue** [reply]
> 这里好壮观啊！

dada有没有看到我的project？
[把我建造的Minecraft进行NFT化](https://github.com/qihuju/qihuju.github.io/projects/3)
我还把我的Minecraft做成脑图了
「点击查看“岳路平脑图个展”：https://qihuju.wiki/docs/qihuju-links/yueluping.html 」
脑图里还有你的神经契约，还有我搭建的museum of dada art

**Comment by Lupingyue** [comment]
截止4月8日晚上8点「北京时间」的脑图

![【奇忽局13】驰骋在比特荒野上的岛屿与河流——漫谈去中心化城市X车联网【🎤录音房】](https://user-images.githubusercontent.com/25149468/114022773-b2fe4980-98a4-11eb-80df-85dded0cfec1.jpg)

**Comment by dadabit** [comment]
On Fri, Apr 9, 2021 at 9:36 AM Lupingyue ***@***.***> wrote:

> OK
>
> 欢迎👏👏，是Dada，还是Dada daddy😄
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/qihuju/qihuju.github.io/discussions/29#discussioncomment-587387>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARYQ3AQYZTBIOTGLVCPPPJLTHZK27ANCNFSM42OA6YKQ>
> .
>
dada

>

**Reply by Lupingyue** [reply]
> 这里好壮观啊！

dada有没有看到我的project？
[把我建造的Minecraft进行NFT化](https://github.com/qihuju/qihuju.github.io/projects/3)
我还把我的Minecraft做成脑图了
「点击查看“岳路平脑图个展”：https://qihuju.wiki/docs/qihuju-links/yueluping.html 」
脑图里还有你的神经契约，还有我搭建的museum of dada art

**Comment by dadabit** [comment]
思维导图很壮观

**Reply by Lupingyue** [reply]
那你应该看这十张图 #21

**Reply by Lupingyue** [reply]
这些思维导图都是可以协同编辑的

**Comment by dadabit** [comment]
您有没有听说过discord

**Reply by Lupingyue** [reply]
> 您有没有听说过discord
> […](#)

我有discord

**Comment by dadabit** [comment]
Open source 😊
😂

**Reply by Lupingyue** [reply]
嗯嗯，一切都可以开源，一切都应该开源

**Comment by dadabit** [comment]
https://discord.gg/CJmQ4ZKy

**Comment by dadabit** [comment]
呢么……如何加入呢？

**Reply by Lupingyue** [reply]
你已经加入了，就先从Minecraft server开始
